broker/src/main/java/io/camunda/zeebe/broker/system/partitions/ZeebePartitionAdminAccess.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker.system.partitions;
+
+import static java.util.Objects.requireNonNull;
+
+import io.camunda.zeebe.broker.Loggers;
+import io.camunda.zeebe.broker.partitioning.PartitionAdminAccess;
+import io.camunda.zeebe.util.sched.ConcurrencyControl;
+import io.camunda.zeebe.util.sched.future.ActorFuture;
+import io.camunda.zeebe.util.sched.future.CompletableActorFuture;
+import java.io.IOException;
+import org.slf4j.Logger;
+
+class ZeebePartitionAdminAccess implements PartitionAdminAccess {
```

Comment:
:wrench: Adding this here since it applies to most methods, but I would not log the errors in this class - we return a future, and the caller should take care of handling the error (including logging). Otherwise you get strange double logging, and often are missing context. We should only log at the source if there's additional context we want to add (and in that case it would not be ERROR level). At the same time, I understand this is only code you moved, so I'm not going to insist at this moment.

Reply:
I would leave it for now.

Logging at the source also has the advantage that you can be sure it is logged, regardless of where it is called from. Yes, double logging is inconvenient, but less so than forgetting somewhere to log at all. Also, this is not a high frequency call site. Also, in this instance, the call sites do not log the errors.

Maybe another addition to the style guide? Seems you have some vision of where logging should happen.